src/doc/rustc/src/platform-support.md

```diff
@@ -134,7 +134,7 @@ target | std | host | notes
 `x86_64-fuchsia` | ✓ |  | 64-bit Fuchsia
 `x86_64-linux-android` | ✓ |  | 64-bit x86 Android
 `x86_64-rumprun-netbsd` | ✓ |  | 64-bit NetBSD Rump Kernel
-`x86_64-sun-solaris` | ✓ |  | 64-bit Solaris 10/11, illumos
+`x86_64-pc-solaris` | ✓ |  | 64-bit Solaris 10/11, illumos
```

Comment:
Doesn't this require changes to CI? Right now we produce artifacts for `x86_64-sun-solaris`.

Reply:
You are right. The previous target will still work, but the artifacts produced should be for the preferred/correct target. I will look into it.

Reply:
Hopefully, I changed it correctly, although I am not a Docker user, so maybe I missed/misunderstood something.